src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -14467,7 +14467,18 @@ public InterceptsLocationAttribute(string filePath, int line, int character)
                 """;
             var generator = new SingleFileTestGenerator(generatedSource, "Generated.cs");
 
-            VerifyOutput(dir, src, includeCurrentAssemblyAsAnalyzerReference: false, additionalFlags: ["/langversion:preview", "/out:embed.exe", "/features:InterceptorsNamespaces=Generated"], generators: [generator], analyzers: null);
+            // Future note: it would be good to have end-to-end tests using InterceptableLocation APIs.
```

Comment:
>Future note [](http://example.com/codeflow?start=15&length=11)

Would it make sense to reference [https://github.com/dotnet/roslyn/issues/72855](https://github.com/dotnet/roslyn/issues/72855) here, so that we consider adding end-to-end tests before closing that issue?

Reply:
Is `EndToEndTests.Interceptors()` that test?

Reply:
`EndToEndTests.Interceptors` is more of a load test, while these are more like integration tests (does configuration flow properly thru the command line, into CommonCompiler, into generators, and allow producing a valid final compilation).

The final version of the tests in this file would likely look a little more like real world versions of the "Usage Examples" section of #72133.

Reply:
Also, we will be prompted to revisit these tests when support for the path-based attributes is dropped, since the tests will stop working completely. Happy to add the link, but I am also not worried about forgetting about them.